src/community/config/tools.py

```diff
@@ -64,6 +66,16 @@ class CommunityToolName(StrEnum):
         category=Category.Function,
         description="Evaluate arithmetic expressions.",
     ),
+    CommunityToolName.Elastic_Search: ManagedTool(
+        name=CommunityToolName.Elastic_Search,
+        implementation=ElasticSearchRetriever,
+        kwargs={"deployment_name": CommunityToolName.Elastic_Search},
+        is_visible=True,
+        is_available=ElasticSearchRetriever.is_available(),
+        error_message="The host and index env variables are required.",
```

Comment:
Just clarify the exact variables eg `ELASTICSEARCH_HOST` and `ELASTICSEARCH_INDEX` here